demos/python_demos/im2latex_demo/README.md

```diff
@@ -0,0 +1,105 @@
+* `tgt` - previous token (for the first time it is `START_TOKEN` )
+Second model is being executed until current decoded token is `END_TOKEN` or length of the formula is less then `--max_formula_len` producing one token per each decode step.
+
+As input, the demo application takes a path to a single image file with a command-line argument `-i`.
+
+The demo workflow is the following:
+
+1. The demo application reads images one by one, crops and pads them to fit into the input image blob of the network (`imgs`). Crop and pad is used to keep size of the font.
+2. For each image, encoder extracts features from the image
+3. While length of the current formula is less then `--max_formula_len` or current token is not `END_TOKEN` Decode Step produces new tokens.
+5. The demo prints the decoded text into the console or in a file if `-o` parameter specified. 
+
+> **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html).
+
+## Running
+
+Run the application with the `-h` option to see the following usage message:
+
+```
+usage: im2latex_demo.py [-h] -m_encoder ENCODER -m_dec_step DEC_STEP -i INPUT
+                        [INPUT ...] [-o OUTPUT_FILE] [-l CPU_EXTENSION]
+                        [--vocab_path VOCAB_PATH] --target_shape TARGET_SHAPE
+                        [TARGET_SHAPE ...] [--max_formula_len MAX_FORMULA_LEN]
+                        [-d DEVICE] [-pf]
+
+Options:
+  -h, --help            Show this help message and exit.
+  -m_encoder M_ENCODER     Required. Path to an .xml file with a trained encoder
+                        part of the model
+  -m_dec_step M_DEC_STEP   Required. Path to an .xml file with a trained decoder
+                        step part of the model
+  -i INPUT [INPUT ...], --input INPUT [INPUT ...]
+                        Required. Path to a folder with images or path to an
+                        image files
+  -o OUTPUT_FILE, --output_file OUTPUT_FILE
+                        Optional. Path to file where to store output. If not
+                        mentioned, result will be storedin the console.
+  -l CPU_EXTENSION, --cpu_extension CPU_EXTENSION
+                        Optional. Required for CPU custom layers. Absolute
+                        MKLDNN (CPU)-targeted custom layers. Absolute path to
+                        a shared library with the kernels implementations
+  --vocab_path VOCAB_PATH
+                        Path to vocab file to construct meaningful phrase
```

Comment:
Is this key **required** or **optional**?

Reply:
Set as required